flags.go

```diff
@@ -37,7 +37,7 @@ const (
 
 	// MetadataPrefetchOnMountFlag is the name of the commandline flag for enabling
 	// metadata-prefetch mode aka 'ls -R' during mount.
-	MetadataPrefetchOnMountFlag = "metadata-prefetch-on-mount"
+	MetadataPrefetchOnMountFlag = "experimental-metadata-prefetch-on-mount"
```

Comment:
I would suggest using ExperimentalMetadataPrefetchOnMountFlag as name to maintain consistency.

Reply:
changed everywhere.